.gitignore

```diff
@@ -193,6 +193,7 @@ build/bin/sage-build-env-config
 /pkgs/*/*.egg-info
 /pkgs/*/.tox
 
+/pkgs/sage-conf/pyproject.toml
 /pkgs/sage-conf_pypi/sage_root/config.log
```

Comment:
Does this need to be here? There is the same item in `pkgs/sage-conf/.gitignore`?

Reply:
Yes, we want all ignore items in `SAGE_ROOT/.gitignore` because this file also doubles as a `.dockerignore` file. 

I do not recall the exact reason why some of the individual `pkgs/*/` directories additionally have their own `.gitignore` files; but I don't want to touch it in this PR.

Reply:
OK.